components/sup/src/manager/service_updater.rs

```diff
@@ -158,7 +158,8 @@ impl ServiceUpdater {
                                                  // census group for our service.
                                                  census_ring: &CensusRing)
                                                  -> Option<PackageIdent> {
-        debug!("Checking for updated package!");
+        debug!("Checking {} for updated {} package in the {} channel",
+               service.bldr_url, service.spec_ident, service.channel);
```

Comment:
🎉

Reply:
This is such a nice touch :+1:

Reply:
![](https://media.giphy.com/media/26gJy8HhekFHkccgM/giphy.gif)